haskell-indexer-backend-ghc/src/Language/Haskell/Indexer/Backend/Ghc.hs

```diff
@@ -456,6 +458,19 @@ refsFromRenamed ctx declAlts (hsGroup, _, _, _) =
             mapMaybe (\(L l n) -> give ctx (nameLocToRef n TypeDecl l)) names
         _ -> []
 
+    refsFromImportDecl :: LImportDecl GhcRn -> [Reference]
+    refsFromImportDecl (L _ idecl) = case idecl of
+      ImportDecl {..} ->
+        case ideclHiding of
+          Nothing -> []
+          Just (_, (L _ imports)) -> mapMaybe refsFromImport imports
+      _ -> []
+
+    refsFromImport :: LIE GhcRn -> Maybe Reference
+    refsFromImport (L _ (IEVarCompat (L l n))) =
+        give ctx (nameLocToRef (ieWrappedName n) Ref l)
```

Comment:
Note: Kythe has the "ref/imports"[1] edge kind, which would be more suitable here. Ok to add in follow-up PR though, please leave a TODO or issue.

[1]: https://kythe.io/docs/schema/#refimports